src/components/FeedbackButton/FeedbackButton.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import FeedbackButton from './FeedbackButton';
+
+it('should match snapshot', () => {
+  const component = shallow(
+    <FeedbackButton />
+  );
+
+  expect(component.debug()).toMatchSnapshot();
+});
+
+it('should dim when required', () => {
+  const component = shallow(
+    <FeedbackButton dimmed={true} />
```

Comment:
Super minor `={true}` is redundant. Fun fact, this syntax mimics that of an HTML 'boolean attribute' which explicitly prohibits setting 'true' or 'false'. https://www.w3.org/TR/html5/infrastructure.html#boolean-attributes